themes/bootstrap5/templates/RecordTab/holdingsils/standard.phtml

```diff
@@ -51,6 +63,11 @@
             <span class="icon-link__label"><?=$this->transEsc($checkILLRequest ? 'ill_request_check_text' : 'ill_request_place_text')?></span>
           </a>
         <?php endif; ?>
+        <?php if ($getThisEnabled && !$isOnlineResource): ?>
```

Comment:
A few thoughts on this logic -- what if we only set $getThisURI at the top of the template if Get This is enabled and the item is not an online resource. Then we could simply check for $getThisURI here instead of the more complex condition. It also might be more clear to escape the value as part of the `<a>` tag rather than pre-escaping it up above. If we do pre-escape it, our convention is to add "Esc" to the end of the variable name to indicate that it is already escaped.